packages/grafana-ui/src/components/Card/Card.mdx

```diff
@@ -0,0 +1,277 @@
+import { Meta, Preview, Props } from "@storybook/addon-docs/blocks";
+import { Card } from "./Card";
+import { Button } from '../Button';
+import { IconButton } from '../IconButton/IconButton';
+import promLogo from '../../../../../public/app/plugins/datasource/prometheus/img/prometheus_logo.svg';
```

Comment:
I'm not 100% sure this will allow storybook to be built? Maybe instead you could use something like https://placeholder.com/

Reply:
Just ran `yarn storybook:build` and it was successful, not sure if it's gonna work in CI tho 🤔

Reply:
It will if the build is OK locally. But I still think **we should not reference core** from grafana-ui.